packages/clerk-js/src/core/resources/UserOrganizationInvitation.ts

```diff
@@ -48,6 +48,13 @@ export class UserOrganizationInvitation extends BaseResource implements UserOrga
     super();
     this.fromJSON(data);
   }
+
+  accept = async (): Promise<UserOrganizationInvitation> => {
+    return await this._basePost({
+      path: `/me/organization_invitations/${this.id}/accept`,
```

Comment:
🔧   This endpoint is missing from the FAPI OpenAPI specs!

Reply:
Yes, we are looking to add it when the feature is available to use